Feature/update kommunikation UCs pt data 886 (#173)
gematik/spec-ISiK-Terminplanung: ImplementationGuide/markdown/Zusammenfassung/UseCases.md

# Übersicht
Im Folgenden wird ein grafischer Überblick über möglichst viele in diesem Modul abgedeckten Anwendungsfälle gegeben.
Da es sich um eine Zusammenfassung handelt, werden nur folgende Use Case und dafür hinreichende Funktionen dargestellt:
* Allgemeine und intuitiv verständliche Use Cases.
  * Kombinationen und weitere Details sind möglich.

>   * Übergreifende Use Cases und und ihre Sub Use Cases können in einem separaten Diagram auf den entsprechenden Seiten gefunden werden.

Reviewer: *Diagram**m**

>   * Übergreifende Use Cases und und ihre Sub Use Cases können in einem separaten Diagram auf den entsprechenden Seiten gefunden werden.

Author: ```suggestion
  * Übergreifende Use Cases und und ihre Sub Use Cases können in einem separaten Diagramm auf den entsprechenden Seiten gefunden werden.
```

* Allgemeine und intuitiv Adverse Use Cases. Diese gilt es zu vermeiden.

> * In den Funktionen werde triviale Suchen einer Ressource anhand ihrer eigenen Properties nicht dargestellt, z.B. Suche einer Ressource anhand der ID, Name, Code usw.

Reviewer: *werde**n**

> * In den Funktionen werde triviale Suchen einer Ressource anhand ihrer eigenen Properties nicht dargestellt, z.B. Suche einer Ressource anhand der ID, Name, Code usw.

Author: ```suggestion
* In den Funktionen werden triviale Suchen einer Ressource anhand ihrer eigenen Properties nicht dargestellt, z.B. Suche einer Ressource anhand der ID, Name, Code usw.
```

## Use Case Diagramm

<img src="https://raw.githubusercontent.com/gematik/spec-ISiK-Terminplanung/rc/main-stufe-4/Material/images/diagrams/usecases.svg" alt="Use Case Diagrams" width="90%"/>

## Use Case - Benachrichtigung (Kommunikation)

> Der Use Case rund um die Benachrichtigung, in Folge "Kommunikation", lässt sich ausdifferenzieren in folgende, die nicht alle durch ISiK sinnvollerweise abgedeckt werden kann, sondern wofür ggf. andere Anwendungen geeigneter erscheinen - wie hier dokumentiert: 

Reviewer: Satz schwer verständlich. Sofern inhaltlich korrekt hier ein Alternativvorschlag:

Der Use Case rund um die Benachrichtigung, im Folgenden "Kommunikation" genannt, lässt sich in folgende Szenarien ausdifferenzieren. Sofern ISiK an bestimmten Stellen nicht geeignet erscheint wird nachfolgend eine Alternative vorgeschlagen:

> Der Use Case rund um die Benachrichtigung, in Folge "Kommunikation", lässt sich ausdifferenzieren in folgende, die nicht alle durch ISiK sinnvollerweise abgedeckt werden kann, sondern wofür ggf. andere Anwendungen geeigneter erscheinen - wie hier dokumentiert: 

Author: ```suggestion
Der Use Case rund um die Benachrichtigung, im Folgenden "Kommunikation" genannt, lässt sich in folgende Szenarien ausdifferenzieren. Sofern ISiK an bestimmten Stellen nicht geeignet erscheint wird nachfolgend eine Alternative vorgeschlagen:
```

1. Interne Kommunikation eines Leistungserbringers mit einem anderen Leistungserbringers während der Buchung eines internen Termins:

>     - Absprache des Termins: "Anfrage des Leistungserbringers: Gibt es an diesem Datum noch einen freien Termin?" -> TIM (?)

Reviewer: TIM ist ein geschützter Begriff. Bitte wie folgt ausschreiben: TI-Messenger

>     - Absprache des Termins: "Anfrage des Leistungserbringers: Gibt es an diesem Datum noch einen freien Termin?" -> TIM (?)

Author: ```suggestion
    - Absprache des Termins: "Anfrage des Leistungserbringers: Gibt es an diesem Datum noch einen freien Termin?" -> TI-Messenger (?)
```

>     - Rückfragen: Rückfrage des Leistungserbringers zur Terminanfrage -> TIM (?)

Reviewer: TIM ist ein geschützter Begriff. Bitte wie folgt ausschreiben: TI-Messenger

>     - Rückfragen: Rückfrage des Leistungserbringers zur Terminanfrage -> TIM (?)

Author: ```suggestion
    - Rückfragen: Rückfrage des Leistungserbringers zur Terminanfrage -> TI-Messenger (?)
```
